tests/test_file.py

```diff
@@ -190,177 +184,196 @@ def test_round_trip(self):
-        cls.input_file_names = [f"file_to_index_{i}.pdf" for i in range(5)]
-        cls.test_file_uris = [
-            f"{cls.input_file_location}/{fname}" for fname in cls.input_file_names
-        ]
-
-        cls.namespace, cls.storage_path, cls.acn = groups._default_ns_path_cred()
-        cls.namespace = cls.namespace.rstrip("/")
-        cls.storage_path = cls.storage_path.rstrip("/")
-        cls.destination = (
-            f"{cls.storage_path}/{testonly.random_name('file-indexing-test')}"
-        )
-
-        # Ingest test files for testing
-        cls.ingested_array_uris = file_ingestion.ingest_files_udf(
-            dataset_uri=cls.destination,
-            file_uris=cls.test_file_uris,
-            acn=cls.acn,
-            namespace=cls.namespace,
-        )
-
-        return super().setUpClass()
-
-    @classmethod
-    def tearDownClass(cls) -> None:
-        """Remove index testing residuals"""
-        _cleanup_residual_test_arrays(array_uris=cls.ingested_array_uris)
-        return super().tearDownClass()
-
-    def tearDown(self) -> None:
-        """Cleanup indexing arrays between tests"""
-        groups.delete(self.created_index_uri, recursive=True)
-        # FIXME: Not a nice way to cleanup vector search residuals:
-        _cleanup_residual_test_arrays(
-            array_uris=[
-                f"tiledb://{self.namespace}/object_metadata",
-                f"tiledb://{self.namespace}/updates",
-                f"tiledb://{self.namespace}/shuffled_vectors",
-                f"tiledb://{self.namespace}/shuffled_vector_ids",
-                f"tiledb://{self.namespace}/partition_indexes",
-                f"tiledb://{self.namespace}/partition_centroids",
-            ]
-        )
-        return super().tearDown()
-
-    @unittest.skip("Extremely slow execution times in the CI/CD client")
-    def test_create_and_update_dataset_udf(self):
-        with self.assertLogs(get_logger_wrapper()) as lg:
-            # Create a vector search group with 1 file
-            self.created_index_uri = file_indexing.create_dataset_udf(
-                search_uri=self.input_file_location,
-                index_uri=f"tiledb://{self.namespace}/{self.destination}",
-                config=client.Ctx().config().dict(),
-                max_files=3,
-            )
-            self.assertTrue("Creating dataset" in lg.output[0])
-
-            # Update the group with all the available files
-            file_indexing.create_dataset_udf(
-                search_uri=self.input_file_location,
-                index_uri=self.created_index_uri,
-                config=client.Ctx().config().dict(),
-            )
-            self.assertTrue("Updating reader" in lg.output[1])
-            index_group_info = groups.info(self.created_index_uri)
-            self.assertIsNotNone(index_group_info)
-            self.assertEqual(index_group_info.asset_count, 6)
+# FIXME: Will be fixed with #595 implementation
+# Disable until then.
```

Comment:
Instead of commenting these out, [use `@unittest.skip`](https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures) on the classes.